selfdrive/controls/lib/longitudinal_mpc_lib/long_mpc.py

```diff
@@ -50,15 +50,31 @@
 T_IDXS = np.array(T_IDXS_LST)
 T_DIFFS = np.diff(T_IDXS, prepend=[0.])
 MIN_ACCEL = -3.5
-T_FOLLOW = 1.45
 COMFORT_BRAKE = 2.5
 STOP_DISTANCE = 6.0
 
+
+class Personality:
+  CHILL = 0
+  NORMAL = 1
+  AGGRESSIVE = 2
+
+
+def get_T_FOLLOW(personality=Personality.NORMAL):
+  if personality==Personality.CHILL:
+    return 1.8
+  elif personality==Personality.NORMAL:
+    return 1.45
+  elif personality==Personality.AGGRESSIVE:
+    return 1.2
+  else:
+    raise NotImplementedError("Longitudinal personality not supported")
+
 def get_stopped_equivalence_factor(v_lead):
   return (v_lead**2) / (2 * COMFORT_BRAKE)
 
 def get_safe_obstacle_distance(v_ego):
```

Comment:
Think you need to make a parameter for distance if you want to change this live

Reply:
Yeah will do that